server/service/src/sync/test/mod.rs

```diff
@@ -25,7 +25,7 @@ impl TestSyncIncomingRecord {
         upsert: U,
     ) -> TestSyncIncomingRecord
     where
-        U: Upsert + 'static,
+        U: Upsert + Clone + 'static,
```

Comment:
I think this is still remainder from the trait stuff ? If so :nit to delete please